examples/a2c/a2c_example.py

```diff
@@ -18,10 +18,11 @@
 # Environment
 env_size = 3  # Options are 1,3,5,7,11
 env_name = f"botbowl-{env_size}"
-env_conf = EnvConf(size=env_size, pathfinding=False)
+env_conf = EnvConf(size=env_size, pathfinding=True)
 
 
-make_agent_from_model = partial(A2CAgent, env_conf=env_conf, scripted_func=a2c_scripted_actions)
+make_agent_from_model = partial(A2CAgent, env_conf=env_conf, scripted_func=a2c_scripted_actions,
+                                exclude_pathfinding_moves=False)
```

Comment:
pathfinding should be False by default, I think. Maybe we should set `exclude_pathfinding_moves= not env_conf.pathfinding`

Reply:
done

Reply:
We should be able to remove `exclude_pathfinding_moves` completely now that we found _filter_actions() to be useless, right?